### DISCUSSION TITLE
How to connect the example `docs` and `web` folder? i.e. https://localhost:3000/docs https://localhost:3000/web

### DISCUSSION THREAD

**Discussion by jasonrichdarmawan** [vercel/turborepo#6308] [Help]
Hi, my goal is to connect the `docs` and `web` folder.
1. I wish to access the app from `https://localhost:3000/docs` or `https://localhost:3000/web`.
2. I wish to share state between docs and web, i.e. a counter (1, 2, 3, and so on).

What I did:
1. https://turbo.build/repo/docs/getting-started/create-new

Search for similar discussions:

<img width="1440" alt="image" src="https://github.com/vercel/turbo/assets/63768126/2dc19fab-79bf-4dbf-a2e9-efeefd22499f">

TLDR, I am new to Next.js and Turbo.

Disclaimer:
1. I am an Angular developer.
3. I used to be a React developer.
4. I am new to the Monorepo concept.
    i. I tried Angular Workspace and it works great but I want independent `package.json` per application.
    ii. I also want SSR. Angular Universal does offer SSR but it's relatively new. Next.js on the other hand do not offer something similar to Angular Workspace. This is where Turbo or Module Federation comes in.
5. I see that Module Federation connect between 2 application with `React.lazy()` https://webpack.js.org/concepts/module-federation/
6. I believe Next.js routing is by folder. https://nextjs.org/docs/app/building-your-application/routing/defining-routes

**Comment by mehulkar** [comment]
This is not something Turborepo can help you with, unfortunately. You may be interested in a microfrontends architecture or look into modeling one of your apps as an importable library that can "mount" into your other app at a specific route.
